src/js/dataMapping/dashboard/fileLabels.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable import/prefer-default-export */
+
+export const fileLabels = {
+    A: 'File A',
+    B: 'File B',
+    C: 'File C',
+    'cross-AB': 'Cross: A/B',
+    'cross-BC': 'Cross: B/C',
+    'cross-CD1': 'Cross: C/D1',
+    'cross-CD2': 'Cross: C/D2'
+};
```

Comment:
Add `/* eslint-enable import/prefer-default-export */` below this line to re-enable the linter rule for other files (since they all get compressed together during the build)

Reply:
D'oh! Great catch.